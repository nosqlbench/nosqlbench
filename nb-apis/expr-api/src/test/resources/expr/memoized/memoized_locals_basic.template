{{alpha = 'one'}}
alpha={{@alpha}}
{{alpha = 'two'}}
alphaAgain={{@alpha}}
{{beta == 'first'}}
betaOnce={{@beta}}
{{beta == 'ignored'}}
betaStill={{@beta}}
{{gamma === 'strict'}}
gammaStrict={{@gamma}}
gammaStrictBang={{@gamma!}}
optional={{@missing?}}
