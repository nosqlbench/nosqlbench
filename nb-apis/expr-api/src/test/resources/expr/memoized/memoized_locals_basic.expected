one
alpha=one
two
alphaAgain=two
first
betaOnce=first
first
betaStill=first
strict
gammaStrict=strict
gammaStrictBang=strict
optional=
